### PR TITLE
Polydisperse systems

### PR DESCRIPTION
**Pull request by griessej**
- Calculator for polydisperse systems 
- Analytic computation of Hessian for polydisperse systems
- Bug fix in tests eam_calculator_forces_hessian

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -1,10 +1,13 @@
 Change log
 ==========
 
-v0.6.0 (9Sep20)
+v0.6.0 (10Sep20)
 -----------------------
 
 - Numerical computation of the Hessian matrix 
+- Calculator for polydisperse systems
```
Comment (pastewka) [inline ChangeLog.md:8]: polydisperse systems is unclear, say something about the interaction potential. Is is generic pair potentials?

File: matscipy/calculators/polydisperse/calculator.py [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+try:
+    from scipy.special import factorial2
+    from scipy.sparse import bsr_matrix
+except ImportError:
+    warnings.warn('Warning: no scipy')
+
+###
+
+
+class IPL():
```
Comment (pastewka) [inline matscipy/calculators/polydisperse/calculator.py:47]: Use more explicit names. Call this InversePowerLawPotential - keystrokes are cheap and abbreviations are difficult to understand for the casual reader.

File: tests/polydisperse_calculator.py [hunk 3]
```diff
@@ -0,0 +1,137 @@
+#! /usr/bin/env pytho
```
Comment (pastewka) [inline tests/polydisperse_calculator.py:1]: python -> python

**Review by pastewka — CHANGES_REQUESTED** [review]

**Comment by griessej** [comment]
Fixed all requests.

**Review by pastewka — APPROVED** [review]

**Pull request merged**